populate/src/main/java/bio/terra/pearl/populate/service/PortalPopulator.java

```diff
@@ -162,7 +162,6 @@ public Optional<Portal> findFromDto(PortalPopDto popDto, FilePopulateContext con
     public Portal overwriteExisting(Portal existingObj, PortalPopDto popDto, FilePopulateContext context) throws IOException {
         Set<CascadeProperty> set = new HashSet<>();
         set.add(PortalService.AllowedCascades.STUDY);
-        set.add(EnrolleeService.AllowedCascades.PARTICIPANT_USER);
```

Comment:
This change could cause a bug and break population for users that were in more than one study (like jsalk and newb) so I removed it